managers/asset-operations-manager.js

```diff
@@ -355,11 +356,15 @@ class AssetOperationsManager {
             if (unfinalizedState != null && unfinalizedState !== ZeroHash) {
                 publicAssertionId = unfinalizedState;
             }
+
+            stateFinalized = false;
+        } else if (state === ASSET_STATES.FINALIZED) {
```

Comment:
We can avoid this condition as there are only 2 enum options: LATEST or FINALIZED. Condition on the line 370 is actually checking if state is FINALIZED